readium/navigator/src/main/java/org/readium/r2/navigator/R2BasicWebView.kt

```diff
@@ -596,9 +571,15 @@ internal open class R2BasicWebView(context: Context, attrs: AttributeSet) : WebV
     }
 
     internal fun shouldOverrideUrlLoading(request: WebResourceRequest): Boolean {
-        if (resourceUrl == request.url.toUrl()) return false
```

Comment:
The `if (resourceUrl == request.url.toUrl())` is not needed?

Reply:
I don't think we should give links to the same page a special treatment. Apps might want to intercept them too.

Reply:
If they intercept it you would end up with a blank page. But for simplicity's sake, that's fine by me.